pkgs/development/compilers/elm/default.nix

```diff
@@ -4,20 +4,11 @@
 let
   fetchElmDeps = import ./fetchElmDeps.nix { inherit stdenv lib fetchurl; };
 
-  patchBinwrap = import ./packages/patch-binwrap.nix { inherit lib writeScriptBin stdenv; };
-
-  elmNodePackages =
-    import ./packages/node-composition.nix {
-      inherit nodejs pkgs;
-      inherit (stdenv.hostPlatform) system;
-    };
-
   hsPkgs = haskell.packages.ghc865.override {
     overrides = self: super: with haskell.lib;
       let elmPkgs = rec {
             elm = overrideCabal (self.callPackage ./packages/elm.nix { }) (drv: {
               # sadly with parallelism most of the time breaks compilation
-              # also compilation is slower with increasing number of cores anyway (Tested on Ryzen 7 and i7)
```

Comment:
I've added this comment in past as a note for myself but forgot to remove it. I don't think it's important enough to keep it there.